### PR TITLE
feat: show api docs and version

### PR DESCRIPTION
**Pull request by AtanasKrondev**

### CODE REVIEW

**Comment by marc0olo** [comment]
closes #99

File: partials/appMainNav.vue [hunk 1]
```diff
@@ -43,7 +43,25 @@
...
     </AppNavAccordion>
+    <div class="footer desktop">
+      <a
+        :href="apiDocs"
+        target="_blank"
+      >
+        æternity middleware docs
+      </a>
+      <p>{{ mdwVersion }}</p>
+    </div>
```
Comment (marc0olo) [inline partials/appMainNav.vue:64]:
```suggestion
      <div class="footer mobile">
        <a
          :href="apiDocs"
          target="_blank"
        >
          API docs
        </a>
        <br>
        {{ mdwVersion }}
      </div>
    </AppNavAccordion>
    <div class="footer desktop">
      <a
        :href="apiDocs"
        target="_blank"
      >
        API docs
      </a>
      <br>
      {{ mdwVersion }}
    </div>
```

Comment (marc0olo) [reply partials/appMainNav.vue:64]: I like it more like this :D

Comment (marc0olo) [reply partials/appMainNav.vue:64]: image

Comment (AtanasKrondev) [reply partials/appMainNav.vue]: Yes, its' much better :)

**Pull request merged**